egs/wsj/s5/utils/data/get_subsegment_feats.sh

```diff
@@ -0,0 +1,46 @@
+#! /bin/bash
+
+# Copyright 2016  Johns Hopkins University (Author: Dan Povey)
+#           2016  Vimal Manohar
+# Apache 2.0.
+
+if [ $# -ne 4 ]; then
+  echo "This scripts gets subsegmented_feats (by adding ranges to data/feats.scp) "
+  echo "for the subsegments file. This is does one part of the "
+  echo "functionality in subsegment_data_dir.sh, which additionally "
```

Comment:
The documentation here is not very good.
It doesn't say that this script writes to its stdout.
The `<subsegments>` format is not documented.
The meaning (and units) of `<frame-shift>` and `<frame-overlap>` is not documented; and the purpose of frame-overlap (to ensure that exactly the right number of frames are written, accounting for edge effects) should be mentioned.
It seems to me that it might be more natural to specify '<frame-length>` instead of frame-overlap, since that is how it's supplied in the mfcc config.  Bear in mind that I hope to, in future, start using the option --snip-edges=false, which will simplify a lot of things.  If this script took a snip_edges option (which would nullify the effect of the frame-length/frame-overlap option) it might make it more future-proof.